lib/core/src/lib/form/components/form-field/form-field.component.ts

```diff
@@ -105,9 +108,15 @@ export class FormFieldComponent implements OnInit, OnDestroy {
     }
 
     private updateReactiveFormControlOnFormRulesEvent(instance: any): void {
-        instance?.formService.formRulesEvent.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(() => {
-            instance?.updateReactiveFormControl();
-            this.triggerFormFieldChanged(instance.field);
+        instance?.formService.formRulesEvent.pipe(takeUntilDestroyed(this.destroyRef)).subscribe((event) => {
+            if (
+                event.field.id !== instance.id &&
+                event.type === 'fieldValueChanged' &&
```

Comment:
what about other event types, like invalid/click etc? do we need this type check here?
![image](https://github.com/user-attachments/assets/b8f96884-861b-4334-bc5a-0ca3e3742d99)

Reply:
changed, triggering update only when value changed